changelogs/current.yaml

```
@@ -42,6 +42,9 @@ behavior_changes:
     If there is no entry referred by the filter config name, the canonical filter name
     (e.g., *envoy.filters.http.buffer* for the HTTP buffer filter) will be used for the backwards
     compatibility.
+- area: router
+  change: |
+    :ref:`total_weight <envoy_v3_api_field_config.route.v3.WeightedCluster.total_weight>` could be optional.
```

```suggestion
    Weighted cluster's :ref:`total_weight <envoy_v3_api_field_config.route.v3.WeightedCluster.total_weight>` is now optional. If not set, Envoy will no longer validate that all weights add up to 100.
```

Also, you changed the meaning if 2 fields. Reference both above with similar language. Thank you!

/wait

---

always got error:
```
generated/rst/version_history/v1.23/v1.23.0.rst:37: WARNING: undefined label: envoy_api_field_route.weightedcluster.total_weight
```

how can I fix this?

---

I'm trying to add ref to this https://github.com/envoyproxy/envoy/pull/21892#discussion_r915983739, which seems like a v2 api?

---

You can ignore v2, but you changed the documentation/defaults for 2 v3 fields, both of those should be linked to.